### ISSUE TITLE
Blank screen after boot and Application Error

### ISSUE THREAD

**Issue by itcane-hatem** [ToniKorin/cordova-plugin-autostart#5]
Hi,
When the device reboot, the app starts with a blank screen and got the following error : 
Application Error : The connection to the server was unsuccessful. (file:///android_asset/www/index.html)

**Comment by ToniKorin** [comment]
Hi,
New issue. Based on the provided info your app was launched, but something went wrong after that in the Cordova framework or in your app.  Have you checked the android logs from your device/emulator?

**Comment by itcane-hatem** [comment]
Hi,
When I start the app manually, it works without any bug. The error is occuring only when the app is started automatically after boot. 
I'm using Cordova 5.3.3 / Android 4.1.1
Here is what I found in logs : 
11-06 10:56:21.912: E/webview(1432): Error: WebView.destroy() called while still attached!
11-06 10:56:21.920: D/webviewglue(1432): nativeDestroy view: 0x67505968
11-06 10:56:21.975: D/SystemWebViewEngine(1432): CordovaWebView is running on device made by: Unitech
11-06 10:56:21.983: I/SystemWebViewEngine(1432): Disabled addJavascriptInterface() bridge since Android version is old.
11-06 10:56:41.068: E/CordovaWebViewImpl(1432): CordovaWebView: TIMEOUT ERROR!

This errors occurs only in auto start after boot

**Comment by ToniKorin** [comment]
Is your app loading something over the internet connection during the start-up phase? 
It is usual that the connection is not available right after the boot phase and your app have to handle it e.g. using the cordova-plugin-network-information plugin.

**Comment by itcane-hatem** [comment]
It's not linked to the Network but I have a new track. The phone is locked with a PIN code at startup. I removed this lock and now the application starts normally. When I put the PIN Code lock, the application starts but after unlocking the phone, the application crashes. I can't understand why. It seem not be an issue in the plugin itself but related to Android.

**Comment by ToniKorin** [comment]
It could be still the network connectivity issue. The cellular network is enabled only after the PIN code phase. WLAN is usually available without the unlock.

**Comment by ToniKorin** [comment]
Any news about your problem?

**Comment by ToniKorin** [comment]
State to closed because no comments from the user and  it was very likely the connectivity issue during the start-up phase.
